packages/datadog-instrumentations/src/net.js

```diff
@@ -17,7 +17,7 @@ const errorTCPCh = channel('apm:net:tcp:error')
 
 const connectionCh = channel(`apm:net:tcp:connection`)
 
-addHook({ name: 'net' }, net => {
+addHook({ name: 'node:net' }, (net, version, name) => {
```

Comment:
I think the idea was to use `name` below but it was not actually done?

Reply:
Forgot to revert from a previous change